internal/acceptance/default_namespace_test.go

```diff
@@ -0,0 +1,71 @@
+package acceptance
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/apierr"
+	"github.com/databricks/databricks-sdk-go/service/settings"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccDefaultNamespaceSetting(t *testing.T) {
+	t.Skip("Automatic test disabled until feature is enabled")
```

Comment:
We should probably create a ticket to track this and put a reminder for when we need to enable it otherwise it would be easy to miss.

Reply:
https://databricks.atlassian.net/browse/DECO-6049